ygot/types.go

```diff
@@ -76,6 +79,17 @@ type GoEnum interface {
 	// generated code file. The ygen library generates a static map of
 	// enumeration values that this method returns.
 	ΛMap() map[string]map[int64]EnumDefinition
+	// String provides the string representation of the enum, which will be
+	// the YANG name if it's in its defined range.
+	String() string
+}
+
+func EnumString(e GoEnum, val int64, enumTypeName string) string {
```

Comment:
I wonder whether we should call this something different? We already have [`ygot.EnumName`](https://godoc.org/github.com/openconfig/ygot/ygot#EnumName) which works slightly differently (it can return an `error` in the case of a number of invalid inputs), so it'd be good to keep to clear as to why users would use this function.

Reply:
I renamed it EnumLogString, as the request was that it's hard to read in a log.